### PR TITLE
:bug: fix mobsf deduplication and severity mapping

### PR DESCRIPTION
**Pull request by manuel-sommer**
fixes:

- issue #7936 
- https://github.com/DefectDojo/django-DefectDojo/blob/master/unittests/scans/mobsf/allsafe.json can't be uploaded without throwing an error because of severity "good".

### CODE REVIEW

**Review by mtesauro — APPROVED** [review]
Approved

File: dojo/tools/mobsf/parser.py [hunk 1]
```diff
@@ -372,11 +372,10 @@ def getSeverityForPermission(self, status):
     # Criticality rating
     def getCriticalityRating(self, rating):
         criticality = "Info"
-        if rating == "warning":
+        if rating == "Good":
```
Comment (Maffooch) [inline dojo/tools/mobsf/parser.py:375]:
```suggestion
        if rating in ["warning", "Good"]:
```

**Review by Maffooch — CHANGES_REQUESTED** [review]
There are many instances of a severity of `warning` in other unit tests in the repo

File: dojo/tools/mobsf/parser.py (hunk 1 above)
Comment (manuel-sommer) [reply dojo/tools/mobsf/parser.py:375]: @Maffooch that "warning" here is useless as it is always replaced already:
https://github.com/DefectDojo/django-DefectDojo/blob/eb3cf16bc68f39069df8154eba11e236130b9a6e/dojo/tools/mobsf/parser.py#L137
https://github.com/DefectDojo/django-DefectDojo/blob/eb3cf16bc68f39069df8154eba11e236130b9a6e/dojo/tools/mobsf/parser.py#L146
https://github.com/DefectDojo/django-DefectDojo/blob/eb3cf16bc68f39069df8154eba11e236130b9a6e/dojo/tools/mobsf/parser.py#L162
https://github.com/DefectDojo/django-DefectDojo/blob/eb3cf16bc68f39069df8154eba11e236130b9a6e/dojo/tools/mobsf/parser.py#L172

warning at this place does basically nothing, I have removed it entirely and all unittests passed

**Review by Maffooch — APPROVED** [review]
It is a little absurd there is a check for "warning" in seven places.. Glad everything works :)

**Comment by manuel-sommer** [comment]
I removed the checks for "warning".

**Review by grendel513 — APPROVED** [review]

**Review by cneill — APPROVED** [review]

**Comment by manuel-sommer** [comment]
Could you please merge this?

**Pull request merged**